pkg/pegout.go

```diff
@@ -47,10 +47,10 @@ type RetainedPegoutQuoteDTO struct {
 }
 
 type PegoutCreationDataDTO struct {
-	GasPrice      uint64  `json:"gasPrice" required:"" description:"The gas price used to compute the gas fee"`
-	FeePercentage float64 `json:"feePercentage" required:"" description:"The percentage fee used to compute the call fee"`
-	FixedFee      uint64  `json:"fixedFee" required:"" description:"The fixed fee used to compute the call fee"`
-	FeeRate       float64 `json:"feeRate" required:"" description:"The fee rate used to compute the gas fee"`
+	GasPrice      uint64   `json:"gasPrice" required:"" description:"The gas price used to compute the gas fee"`
```

Comment:
Why is GasPrice kept as a uint64 here while in other places of the codebase has a dynamic value https://github.com/rsksmart/liquidity-provider-server/blob/ad66507faf90dc1a8d4e43b89559f5378d5d84d3/internal/entities/blockchain/rootstock.go#L38